docs/source/x_ext.rst

```diff
@@ -414,7 +423,8 @@ Issue interface
   +------------------------+--------------------------+-----------------------------------------------------------------------------------------------------------------+
   | ``rs[X_NUM_RS-1:0]``   | logic [X_RFR_WIDTH-1:0]  | Register file source operands for the offloaded instruction.                                                    |
   +------------------------+--------------------------+-----------------------------------------------------------------------------------------------------------------+
-  | ``rs_valid``           | logic [X_NUM_RS-1:0]     | Validity of the register file source operand(s).                                                                |
+  | ``rs_valid``           | logic [X_NUM_RXREGS-1:0] | Validity of the register file source operand(s). If register pairs are supported, the validty is signaled for   |
```

Comment:
Can you explain which bit in rs_valid[] corresponds to which bits in rs[]?

Reply:
Good point. I think it is best to define this along the parameter definition, so it will be consistent when used at multiple places. Actually, I think even the mapping of bits in rs is not very clearly defined, but I guess we should look at this in a separate issue.